### PR TITLE
Infinispan Dev UI without Infinispan configuration

### PR DESCRIPTION
**Pull request by karesti**
When the configuration of infinispan is not provided, the Dev UI breaks

Without configuration:

Screenshot 2021-07-29 at 17 06 21

With configuration:

Screenshot 2021-07-29 at 17 06 32

### CODE REVIEW

**Review by rsvoboda — COMMENTED** [review]
+1 for the PR, tried with / without running ISPN, some misconfigurations.

The only comment I have is for `Web Console` link. I was using both `quay.io/infinispan/server` and `quay.io/infinispan/server-native`. For `server-native` there is no web console available. Not sure if it's worth checking the availability of the console.

File: extensions/infinispan-client/runtime/src/main/java/io/quarkus/infinispan/client/runtime/InfinispanServerUrlSupplier.java [hunk 1]
```diff
@@ -6,12 +6,16 @@
...
 
     @Override
-    public RemoteCacheManager get() {
+    public String get() {
         RemoteCacheManager cacheManager = cacheManager();
-        return cacheManager;
+        if (cacheManager == null) {
+            return "";
+        }
+        return cacheManager.getConfiguration().servers().get(0).host() +
```
[the remark this answers is not in the document]
Comment (rsvoboda) [reply extensions/infinispan-client/runtime/src/main/java/io/quarkus/infinispan/client/runtime/InfinispanServerUrlSupplier.java:17]: In which situation would be host and port null ?

**Comment by karesti** [comment]
@rsvoboda yes, the lack of console in the native infinispan server is an issue that needs to be fixed in the infinispan server native but the web console is supposed to be always there cc @wburns

File: extensions/infinispan-client/runtime/src/main/java/io/quarkus/infinispan/client/runtime/InfinispanServerUrlSupplier.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (netodevel) [reply extensions/infinispan-client/runtime/src/main/java/io/quarkus/infinispan/client/runtime/InfinispanServerUrlSupplier.java]: I don't know, just   seems easy code to have an NPE.

**Comment by geoand** [comment]
Seems like a formatting change is needed

**Comment by wburns** [comment]
> @rsvoboda yes, the lack of console in the native infinispan server is an issue that needs to be fixed in the infinispan server native but the web console is supposed to be always there cc @wburns

I never trust anything is working with the ISPN native server, but what we test in CI. So I would say it is the Schrödinger's console :)

**Comment by gsmet** [comment]
Rebased and fixed the formatting issue.

**Review by gsmet — APPROVED** [review]

**Comment by karesti** [comment]
@gsmet thanks for the formatting

**Pull request merged**